dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -708,13 +706,16 @@ private void verify(List<String> args) {
             if (containsHelpRequest(args) || args.isEmpty()) {
                 verifyHelp();
             } else {
+                /*
                 String entry = reqVal(args, "--entry");
                 String version = reqVal(args, "--version");
                 String verifySource = optVal(args, "--verified-source", null);
 
                 final boolean unverifyRequest = args.contains("--unverify");
                 final boolean isScript = SCRIPT.get();
                 handleVerifyUnverify(entry, version, verifySource, unverifyRequest, isScript);
+                 */
+                out("This command is currently deprecated. Use the extended TRS webservice endpoint instead.");
```

Comment:
I would say "This command is deprecated". The word "currently" implies it might be coming back (or is it coming back?).

Reply:
It should come back